Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Actions.swift

```diff
@@ -62,13 +62,11 @@ extension ZMConversationMessage {
         guard conversation.conversationType == .group else {
             return false
         }
-
+        
         // Show the message details in Team groups.
-        if ZMUser.selfUser()?.isTeamMember == true {
+        if conversation.teamRemoteIdentifier != nil {
             return true
```

Comment:
Maybe it would actually be safer to do more checks in the case of team users. For example, returning true here means we would show the details for a ping that the user didn't send.

```suggestion
            return canBeLiked || isSentBySelfUser
```